providers/efa/man/efadv_query_device.3.md

```diff
@@ -76,6 +76,9 @@ struct efadv_device_attr {
 		Reading source address (SGID) from receive completion descriptors is supported.
 		Valid only for unknown AH.
 
+	EFADV_DEVICE_ATTR_CAPS_RDMA_WRITE:
+		RDMA write is supported
+
 *max_rdma_size*
```

Comment:
Does this apply to both RDMA read and write now?

Reply:
Hey,
No this does not apply for RDMA read. For RDMA read you must get also EFADV_DEVICE_ATTR_CAPS_RDMA_READ to know it is supported.

Reply:
I'm referring to max_rdma_size.

Reply:
Hey, sorry.
Yes, the max RDMA size is identical for read and write.